README.md

```diff
@@ -1,6 +1,6 @@
 ![Electron Installer for Windows](resources/logo.png)
 
-# electron-installer-windows [![Version](https://img.shields.io/npm/v/electron-installer-windows.svg)](https://www.npmjs.com/package/electron-installer-windows) [![Build Status](https://img.shields.io/travis/unindented/electron-installer-windows.svg)](http://travis-ci.org/unindented/electron-installer-windows) [![Build Status](https://img.shields.io/appveyor/ci/unindented/electron-installer-windows.svg)](https://ci.appveyor.com/project/unindented/electron-installer-windows) [![Dependency Status](https://img.shields.io/gemnasium/unindented/electron-installer-windows.svg)](https://gemnasium.com/unindented/electron-installer-windows)
+# electron-installer-windows [![Version](https://img.shields.io/npm/v/electron-installer-windows.svg)](https://www.npmjs.com/package/electron-installer-windows) [![Build Status](https://img.shields.io/travis/electron-userland/electron-installer-windows.svg?logo=travis)](http://travis-ci.org/electron-userland/electron-installer-windows) [![Build Status](https://img.shields.io/appveyor/ci/electron-userland/electron-installer-windows.svg?logo=appveyor)](https://ci.appveyor.com/project/electron-userland/electron-installer-windows)
```

Comment:
Linux/macOS Build Status, Windows Build Status vs. two "Build Status" alt tags.

Reply:
I copied how `electron-winstaller` module which only differentiates builds based on icons, I'll change the text inside them, it'll be clearer that way

Reply:
`electron-winstaller`? As far as I can tell, that differentiates based on service name:

https://github.com/electron/windows-installer/blame/736159e2808d553ae049112cbd67f939a647122a/README.md#L3-L4

Reply:
The service name is not shown on the rendered README, so even if it's shown as `Windows Build Status` nothing is really shown on the actual README.
I added the labels directly to the image, so it's more readable but it seems it's a little bit too long. I might just remove the `Build Status` and just leave the name of the OSs, everybody knows that the badges are for build status so I don't think that's needed.
Check how it currently looks here and if you agree with removing the words `Build Status` https://github.com/electron-userland/electron-installer-windows/tree/fix-links#electron-installer-windows---